### DISCUSSION TITLE
下载.exe file设置为管理员模式启动后，打开网易云音乐，discord个人状态并未同步音乐状态

### DISCUSSION THREAD

**Discussion by Habuyoshiharu** [aliencaocao/netease_cloudmusic_discord_rpc#20] [Q&A] [answered]
我的启动顺序是 梯→Dc→exe文件管理员启动→网易云音乐
下图是网易云版本截图 官网直接下载的软件
![rrrr](https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/assets/134794289/22435848-3244-4972-bc1e-0de5102803d4)

**Comment by aliencaocao** [comment]
右键管理员运行不够，必须要进到属性里把管理员运行打钩。这两者不是一个效果

**Reply by Habuyoshiharu** [reply]
是的，我是直接设置的exe属性为管理员运行后启动的

**Reply by aliencaocao** [reply]
我这边无法复现问题。
请在任务管理器里退出rpc，然后在exe旁边创建一个空白文件debug.log，然后再尝试按照你说的启动。debug.log文件里会记录日志，等你复现问题后把里面的日志发给我

**Reply by Habuyoshiharu** [reply]
> 我这边无法复现问题。 请在任务管理器里退出rpc，然后在exe旁边创建一个空白文件debug.log，然后再尝试按照你说的启动。debug.log文件里会记录日志，等你复现问题后把里面的日志发给我

好的 我试一下

**Reply by Habuyoshiharu** [reply]
> 我这边无法复现问题。 请在任务管理器里退出rpc，然后在exe旁边创建一个空白文件debug.log，然后再尝试按照你说的启动。debug.log文件里会记录日志，等你复现问题后把里面的日志发给我

按照您说的尝试之后debug中未记录任何日志且后台的RPC也在运行，但是不起效果，我有时间在尝试吧，感谢！

**Reply by Habuyoshiharu** [reply, marked as answer]
> 我这边无法复现问题。 请在任务管理器里退出rpc，然后在exe旁边创建一个空白文件debug.log，然后再尝试按照你说的启动。debug.log文件里会记录日志，等你复现问题后把里面的日志发给我

感谢您的回复！我发现了问题是我没有在设置中开启“当前隐私状态”

**Reply by aliencaocao** [reply]
如果感觉好用还请加个星，谢谢
